packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -1471,7 +1471,7 @@ export interface CallWithChatControlOptions extends CommonCallControlOptions {
 }
 
 // @public
```

Comment:
i don't know why but there was an extra space which is removed when i run rush update: 
<img width="1112" alt="image" src="https://github.com/Azure/communication-ui-library/assets/109105353/7c6bf6c8-d8ae-4f59-9bf6-99a6fc6e704c">